test/library/standard/FileSystem/chmod/changeMode.execopts

```diff
@@ -0,0 +1,19 @@
+--permissions 0o0600 # changeMode.rwu.good
+--permissions 0o0400 # changeMode.ru.good
+--permissions 0o0200 # changeMode.wu.good
+--permissions 0o0100 # changeMode.xu.good
+--permissions 0o0700 # changeMode.rwxu.good
+--permissions 0o0040 # changeMode.rg.good
+--permissions 0o0020 # changeMode.wg.good
+--permissions 0o0010 # changeMode.xg.good
+--permissions 0o0070 # changeMode.rwxg.good
+--permissions 0o0004 # changeMode.ro.good
+--permissions 0o0002 # changeMode.wo.good
+--permissions 0o0001 # changeMode.xo.good
+--permissions 0o0007 # changeMode.rwxo.good
+--permissions 0o4000 # changeMode.su.good
+--permissions 0o2000 # changeMode.sg.good
+--permissions 0o1000 # changeMode.to.good
+--permissions 0o4700 # changeMode.rwsu.good
+--permissions 0o2070 # changeMode.rwsg.good
+--permissions 0o1007 # changeMode.rwto.good
```

Comment:
Nice!  I'd be tempted to leave the non-octal settings as well, to demonstrate that it continues to work.  They'd be documented by the name of the expected .good file